src/storybook/decorators.tsx

```diff
@@ -8,6 +8,31 @@ import { ApolloProvider } from '@apollo/client'
 import { ApolloClient, InMemoryCache } from '@apollo/client'
 import { useForm, FormProvider, UseFormOptions } from 'react-hook-form'
 
+import { colors } from 'styles/theme'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faMoon, faSun } from '@fortawesome/free-regular-svg-icons'
+
+import { ThemeProvider } from 'styled-components'
+
+export const DarkModeThemeToggler = (DecoratedStory: () => JSX.Element): JSX.Element => {
+  const [darkMode, setDarkMode] = React.useState(false)
+  const theme = colors(darkMode)
+
+  const handleDarkMode = React.useCallback(() => setDarkMode((darkMode) => !darkMode), [])
```

Comment:
No need to memoize here, as you pass it not to a custom Component but to `<button>` which rerenders anyway all the time

Reply:
Ah ok, cool thx

Reply:
Ah ok, cool thx